src/legacy/core_plugins/embeddable_api/public/np_ready/public/lib/actions/action.test.ts

```diff
@@ -17,23 +17,20 @@
  * under the License.
  */
 
-import '../ui_capabilities.test.mocks';
-jest.mock('ui/new_platform');
-
-import { HelloWorldAction, SayHelloAction, EmptyEmbeddable } from '../test_samples/index';
+import { SayHelloAction } from '../test_samples/actions/say_hello_action';
+import { HelloWorldAction } from '../test_samples/actions/hello_world_action';
+import { EmptyEmbeddable } from '../test_samples/embeddables/empty_embeddable';
```

Comment:
> The benefit I see of importing from index is that you can move things around internally to the `test_samples` folder without having to change imports elsewhere.

Yes, totally, I'm thinking maybe to add `index.ts` back.

> Maybe there are better reasons though I'm unaware of!

Yes, I had the following reason: there were couple of dozen files in that folder; as I refactored code, the test samples would temporarily break, and because all of them were exported from one `index.ts` top level file no tests would work and TypeScript would complain. Even if a test depended on a single file from the test samples folder, but some unrelated file in that folder was broken, you could not run the tests. Thus, I imported in tests only explicitly what those tests need, to be able to fix those tests and test samples.

Reply:
I've added back `index.ts` for ease of importing in the future.